server/app/views/admin/programs/ProgramBlocksView.java

```diff
@@ -164,7 +174,14 @@ public Content render(
                                 div()
                                     .withClasses("flex")
                                     .with(renderEditButton(request, programDefinition))
-                                    .with(renderPreviewButton(programDefinition))),
+                                    .with(renderPreviewButton(programDefinition)))
+                            .with(
+                                iff(
+                                    malformedQuestionDefinition,
+                                    div(
+                                        p("Some questions are not pointing at the latest version."
+                                                + " Edit the program and try republishing.")
```

Comment:
**suggestion (blocking):** Instruct the user to file a bug if they see this message

Reply:
done